lib/matplotlib/axis.py

```diff
@@ -1743,6 +1750,12 @@ def convert_units(self, x):
                                          f'units: {x!r}') from e
         return ret
 
+    def _get_shared_axis(self):
+        """Return list of shared axis."""
+        axisnameattr = f"{self._get_axis_name()}axis"
+        return [getattr(ax, axisnameattr) for ax in
```

Comment:
Instead of going through getattr you can just key off _axis_map?

Reply:
Seems possible now that I understand the code a bit better (as opposed to just reorganizing it...).